installer/darwin/eula_resources_template.xml

```diff
@@ -0,0 +1,110 @@
+<!--
+This template is taken from the create-dmg utility
+https://github.com/create-dmg/create-dmg
```

Comment:
Copied from this file and hard coded the EULA_FORMAT: https://github.com/create-dmg/create-dmg/blob/master/support/eula-resources-template.xml